Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.70 AS builder
+FROM rust:1.73 AS builder
```

Comment:
In the meantime I found a better long-term solution: Can you replace `rust:1.73` with `rust:1.73-bullseye` ? That pins the underlying Debian version for the builder image to one that works with our `ghidra_headless_base` image.